content/participants/ttjc@RahulChugwani/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: RahulChugwani
+type: "participant"
+name: "Rahul Chugwani"
+portfolio: "https://tanaypratap.com"
```

Comment:
Kindle change the portfolio link with your portfolio or make it blank if not made yet.